CODIGO/Declares.bas

```diff
@@ -395,6 +395,23 @@ Public TargetYMacro            As Byte
 ' Constantes de intervalo
 Public Const INT_MACRO_HECHIS  As Integer = 500
 Public Const INT_MACRO_TRABAJO As Integer = 1200
+
+Public Type t_Intervals
```

Comment:
Would it not better to move this into a class to avoid over population in Declares.bas?

Reply:
having simple declarations here avoid a lot of cross reference issues, also we are not declaring anything new but moving global vars into a struct to make it easy to pass it to c++